README.md

```diff
@@ -45,6 +45,37 @@ Or if you want paid support please contact Catalyst IT Australia:
 https://www.catalyst-au.net/contact-us
 
 
+3. Set up master password feature for load test (Non Production Environments)
```

Comment:
Break this out into a separate heading, and add a link to it in the table of contents

I would put this under the 'Curl example' heading